lib/Client.php

```diff
@@ -304,6 +311,20 @@ public function setRetryOnLimit($retry)
         return $this;
     }
 
+    /**
+     * Set default verify certificates flag
+     *
+     * @param bool $verifySSLCerts
+     *
+     * @return Client
+     */
+    public function setVerifySSLCerts($verifySSLCerts)
```

Comment:
It only has the `setVerifySSLCerts` method.

Do we need to add the `getVerifySSLCerts` method?

I think we have this method to get current `$verifySSLCerts` value.

Reply:
I think it's fine as-is. We don't have getters for the other options.